common/src/KokkosFFT_utils.hpp

```diff
@@ -118,14 +118,12 @@ bool is_out_of_range_value_included(const ContainerType& values, IntType max) {
 template <
     typename ViewType, template <typename, std::size_t> class ArrayType,
     typename IntType, std::size_t DIM = 1,
-    std::enable_if_t<std::is_integral_v<IntType>, std::nullptr_t> = nullptr>
+    std::enable_if_t<Kokkos::is_view_v<ViewType> && std::is_integral_v<IntType>,
+                     std::nullptr_t> = nullptr>
```

Comment:
Do you have other overloads for this function ? If not I would simply use a static assertion instead of SFINAE.

Reply:
True. I have fixed accordingly.